Sources/BraintreeVenmo/BTVenmoClient.swift

```diff
@@ -31,6 +31,9 @@ import BraintreeCore
     /// This allows us to set and return a completion in our methods that otherwise cannot require a completion.
     var appSwitchCompletion: (BTVenmoAccountNonce?, Error?) -> Void = { _, _ in }
 
+    /// True if `tokenize()` was called with a Vault request object type
+    var isVaultRequest: Bool = false
```

Comment:
Looks like we already have a property for this, `shouldVault` on line 28 we can re-use and just pass into our analytics calls

Reply:
➕ 1

Reply:
oh wow completely missed that variable. thanks!